test/models/onnx/flatten.prototxt

```diff
@@ -21,20 +21,6 @@ graph {
     type {
       tensor_type {
         elem_type: 1
-        shape {
-          dim {
-            dim_value: 1
-          }
-          dim {
-            dim_value: 2
-          }
-          dim {
-            dim_value: 2
-          }
-          dim {
-            dim_value: 2
-          }
-        }
```

Comment:
It would be probably better to create a new test with a new model instead of modifying the existing one.

Reply:
In this test was bug (I am not sure how it works util now), Flatten always return 2D output shape